src/plugin.js

```diff
@@ -231,18 +260,33 @@ const onPlayerReady = (player) => {
  * to you; if not, remove the wait for "ready"!
  *
  * @function eme
+ * @param    {Object} [player]
+ *           A player object.
  * @param    {Object} [options={}]
  *           An object of options left to the plugin author to define.
  */
-const eme = function(options = {}) {
-  this.eme.options = options;
-
-  this.ready(() => onPlayerReady(this));
+const eme = function(player, options = {}) {
+  player.ready(() => onPlayerReady(player));
+
+  player.eme = {
+    initializeMediaKeys(emeOptions = {}) {
+      const mergedEmeOptions = videojs.mergeOptions(
+        player.currentSource(),
+        options,
+        emeOptions
+      );
+
+      return initializeMediaKeys(player, mergedEmeOptions);
```

Comment:
can we just change initializeMediaKeys to take the various option objects and do the merging in the function?